src/app/app.component.html

```diff
@@ -1,13 +1,17 @@
-<div class="container text-center my-4">
-  <!-- 🍺🥃☠ -->
-  <h1 class="display-4 font-monospace" style="font-variant: small-caps;">Ketal</h1>
-  <p class="lead">Grosse Guinze Generator</p>
-  <!-- Display the player choices -->
-  <app-players-list *ngIf="displayPlayerChoice" (onBeginGame)="beginGame()"></app-players-list>
-  <!-- Guinze -->
-  <app-game *ngIf="!displayPlayerChoice" (goToPhaseTwo)="beginSecondPhase()"></app-game>
-  <app-second-phase *ngIf="displaySecondPhase"></app-second-phase>
-</div>
-
-
-
+<app-header></app-header>
+
+<div class=" main container text-center my-4">
+    <!-- Display the player choices -->
+    <!--    <app-players-list *ngIf="displayPlayerChoice" (onBeginGame)="beginGame()"></app-players-list>-->
+    <app-players-list *ngIf="!this.gameSrv.game"></app-players-list>
+    <!-- Guinze -->
+    <app-game *ngIf="this.gameSrv.game"
+              (goToPhaseTwo)="beginSecondPhase()"></app-game>

```

Comment:
it seems that the event does not exist anymore
```suggestion
    <app-game *ngIf="this.gameSrv.game"></app-game>

Reply:
Bien vu je l'avais oublié